pkg/ingester/index/index_test.go

```diff
@@ -25,26 +25,10 @@ func Test_GetShards(t *testing.T) {
 		{16, &astmapper.ShardAnnotation{Shard: 15, Of: 16}, []uint32{15}},
 
 		// idx factor a larger multiple of schema factor
-		{32, &astmapper.ShardAnnotation{Shard: 0, Of: 16}, []uint32{0, 1}},
-		{32, &astmapper.ShardAnnotation{Shard: 4, Of: 16}, []uint32{8, 9}},
-		{32, &astmapper.ShardAnnotation{Shard: 15, Of: 16}, []uint32{30, 31}},
-		{64, &astmapper.ShardAnnotation{Shard: 15, Of: 16}, []uint32{60, 61, 62, 63}},
-
-		// schema factor is a larger multiple of idx factor
-		{16, &astmapper.ShardAnnotation{Shard: 0, Of: 32}, []uint32{0}},
-		{16, &astmapper.ShardAnnotation{Shard: 4, Of: 32}, []uint32{2}},
-		{16, &astmapper.ShardAnnotation{Shard: 15, Of: 32}, []uint32{7}},
-
-		// idx factor smaller but not a multiple of schema factor
-		{4, &astmapper.ShardAnnotation{Shard: 0, Of: 5}, []uint32{0}},
-		{4, &astmapper.ShardAnnotation{Shard: 1, Of: 5}, []uint32{0, 1}},
-		{4, &astmapper.ShardAnnotation{Shard: 4, Of: 5}, []uint32{3}},
-
-		// schema factor smaller but not a multiple of idx factor
-		{8, &astmapper.ShardAnnotation{Shard: 0, Of: 5}, []uint32{0, 1}},
-		{8, &astmapper.ShardAnnotation{Shard: 2, Of: 5}, []uint32{3, 4}},
-		{8, &astmapper.ShardAnnotation{Shard: 3, Of: 5}, []uint32{4, 5, 6}},
-		{8, &astmapper.ShardAnnotation{Shard: 4, Of: 5}, []uint32{6, 7}},
```

Comment:
We don't have a check for `Of` to be a multiple of `totalShards`. Do we want to keep these tests or add a check for it?

Reply:
I don't think we need those. Ingester and storage uses respectively always 16 and 32.  I don't think the storage will change soon enough.